subsys/usb/class/netusb/function_ecm.c

```diff
@@ -31,6 +31,7 @@ LOG_MODULE_REGISTER(usb_ecm);
 
```

Comment:
In commit `usb: ECM fix to pass USB3CV tests`
Could we just say `Fixes #24200` instead of the longer string.

Reply:
Yes, no problem.